test/core/node2code/undo/regress7864.dyn

```diff
@@ -0,0 +1,24 @@
+<Workspace Version="0.8.2.1957" X="0" Y="0" zoom="1" Name="Home" RunType="Automatic" RunPeriod="1000" HasRunWithoutCrash="True">
```

Comment:
Node to code undo test framework will automatically pick up this file and do node to code and undo operations, so no need to write a testcase.